### PR TITLE
https://issues.redhat.com/browse/ACM-6119

### PR DESCRIPTION
**Pull request by faaden123**

### CODE REVIEW

File: networking/submariner/install_subctl.adoc [hunk 1]
```diff
@@ -44,3 +44,8 @@ After adding the utility to your path, view the following table for a brief desc
 |===
 
 For more information about the `subctl` utility and its commands, see link:https://submariner.io/operations/deployment/subctl/[`subctl` in the Submariner documentation].
+
+*Note*: Only the commands listed above support `subctl`.
```
Comment (faaden123) [inline networking/submariner/install_subctl.adoc]:
```suggestion
*Note*: Only the the most relevant commands, including those listed above, support `subctl`. 
```

Comment (faaden123) [reply networking/submariner/install_subctl.adoc:48]: This is a suggestion following @dfarrell07 suggestion from the previous PR.

File: networking/submariner/install_subctl.adoc [hunk 2]
```diff
@@ -44,3 +44,8 @@ After adding the utility to your path, view the following table for a brief desc
 |===
 
 For more information about the `subctl` utility and its commands, see link:https://submariner.io/operations/deployment/subctl/[`subctl` in the Submariner documentation].
+
+*Note*: Only the commands listed above support `subctl`. 
+
+For more information about the `subctl` utility and its commands, see link:https://submariner.io/operations/deployment/subctl/[`subctl` in the Submariner documentation].
```
Comment (dockerymick) [inline networking/submariner/install_subctl.adoc]: Looks like this is a duplicate statement. This statement is also on line 46.

File: networking/submariner/install_subctl.adoc (hunk 1 above)
Comment (dockerymick) [inline networking/submariner/install_subctl.adoc]: Improve this statement for accessibility: https://www.ibm.com/docs/en/ibm-style?topic=medium-accessibility

**Review by dockerymick — CHANGES_REQUESTED** [review]

File: networking/submariner/install_subctl.adoc (hunk 2 above)
Comment (faaden123) [reply networking/submariner/install_subctl.adoc]: Thank you!

File: networking/submariner/install_subctl.adoc [hunk 3]
```diff
@@ -43,4 +43,7 @@ After adding the utility to your path, view the following table for a brief desc
 | link:https://submariner.io/operations/deployment/subctl/#version[`version`] | Displays the version details of the `subctl` binary tool.
 |===
 
+*Note*: Only the commands listed above support `subctl`. 
+
 For more information about the `subctl` utility and its commands, see link:https://submariner.io/operations/deployment/subctl/[`subctl` in the Submariner documentation].
```
Comment (faaden123) [inline networking/submariner/install_subctl.adoc]:
```suggestion

```

File: networking/submariner/install_subctl.adoc [hunk 4]
```diff
@@ -43,4 +43,6 @@ After adding the utility to your path, view the following table for a brief desc
 | link:https://submariner.io/operations/deployment/subctl/#version[`version`] | Displays the version details of the `subctl` binary tool.
 |===
 
-For more information about the `subctl` utility and its commands, see link:https://submariner.io/operations/deployment/subctl/[`subctl` in the Submariner documentation].
+*Note*: Only the commands listed above support `subctl`. For more information about the `subctl` utility and its commands, see link:https://submariner.io/operations/deployment/subctl/[`subctl` in the Submariner documentation].
```
Comment (dockerymick) [inline networking/submariner/install_subctl.adoc]: Improve this statement for accessibility: https://www.ibm.com/docs/en/ibm-style?topic=medium-accessibility

**Review by dockerymick — CHANGES_REQUESTED** [review]

File: networking/submariner/install_subctl.adoc (hunk 4 above)
Comment (dfarrell07) [inline networking/submariner/install_subctl.adoc]:
```suggestion
*Note*: Red Hat's build of `subctl` only includes commands relevant to ACM deployments. For more information about the `subctl` utility and its commands, see link:https://submariner.io/operations/deployment/subctl/[`subctl` in the Submariner documentation].
```

Same feedback as on https://github.com/stolostron/rhacm-docs/pull/5252.

File: networking/submariner/install_subctl.adoc [hunk 5]
```diff
@@ -43,4 +43,6 @@ After adding the utility to your path, view the following table for a brief desc
 | link:https://submariner.io/operations/deployment/subctl/#version[`version`] | Displays the version details of the `subctl` binary tool.
 |===
 
-For more information about the `subctl` utility and its commands, see link:https://submariner.io/operations/deployment/subctl/[`subctl` in the Submariner documentation].
+*Note*: `subctl` can only support commands relevant to {rhacm-short} deployments which are listed above. For more information about the `subctl` utility and its commands, see link:https://submariner.io/operations/deployment/subctl/[`subctl` in the Submariner documentation].
```
Comment (faaden123) [inline networking/submariner/install_subctl.adoc]: {rhacm-short} . Need to locate where the shortened version of product names are in docs.

File: networking/submariner/install_subctl.adoc [hunk 6]
```diff
@@ -43,4 +43,6 @@ After adding the utility to your path, view the following table for a brief desc
 | link:https://submariner.io/operations/deployment/subctl/#version[`version`] | Displays the version details of the `subctl` binary tool.
 |===
 
-For more information about the `subctl` utility and its commands, see link:https://submariner.io/operations/deployment/subctl/[`subctl` in the Submariner documentation].
+*Note*: `subctl` can only support commands relevant to {product-title} deployments which are listed above. For more information about the `subctl` utility and its commands, see link:https://submariner.io/operations/deployment/subctl/[`subctl` in the Submariner documentation].
```
Comment (dockerymick) [inline networking/submariner/install_subctl.adoc]: @dfarrell07 made a comment in the [previous PR](https://github.com/stolostron/rhacm-docs/pull/5252#discussion_r1262822375): 

Screenshot 2023-08-04 at 8 21 53 AM

Improve this statement for accessibility: https://www.ibm.com/docs/en/ibm-style?topic=medium-accessibility

Words like above, below, left, right should not be used in customer product doc. 

And check for guidance on Possessives: https://www.ibm.com/docs/en/ibm-style?topic=grammar-possessives

**Review by dockerymick — CHANGES_REQUESTED** [review]

File: networking/submariner/install_subctl.adoc (hunk 6 above)
Comment (dockerymick) [inline networking/submariner/install_subctl.adoc]:
```suggestion
*Note*: The Red Hat build of `subctl` includes only the commands that are relevant to {product-title}. For more information about the `subctl` utility and its commands, see link:https://submariner.io/operations/deployment/subctl/[`subctl` in the Submariner documentation].
```

**Comment by dockerymick** [comment]
@dfarrell07 @maayanf24 I have made updates to this PR. When you both have an opportunity, will you review it?

File: networking/submariner/install_subctl.adoc [hunk 7]
```diff
@@ -43,4 +43,6 @@ After adding the utility to your path, view the following table for a brief desc
 | link:https://submariner.io/operations/deployment/subctl/#version[`version`] | Displays the version details of the `subctl` binary tool.
 |===
 
-For more information about the `subctl` utility and its commands, see link:https://submariner.io/operations/deployment/subctl/[`subctl` in the Submariner documentation].
+*Note*: The Red Hat build of `subctl` includes only the commands that are relevant to {product-title}. For more information about the `subctl` utility and its commands, see link:https://submariner.io/operations/deployment/subctl/[`subctl` in the Submariner documentation].
```
Comment (cadawson) [inline networking/submariner/install_subctl.adoc]:
```suggestion
*Note*: The Red Hat build of `subctl` only includes the commands that are relevant to {product-title}. For more information about the `subctl` utility and its commands, see link:https://submariner.io/operations/deployment/subctl/[`subctl` in the Submariner documentation].
```

**Review by cadawson — APPROVED** [review]
Looks good. Minor suggestion added. Thanks!

**Review by dfarrell07 — APPROVED** [review]

**Review by maayanf24 — APPROVED** [review]

**Review by oafischer — APPROVED** [review]

**Pull request merged**